### PR TITLE
Javascript/javascript3/week2

### PR DESCRIPTION
**Pull request by Sarulathaanbu**

### CODE REVIEW

File: javascript/javascript3/week3/visual/main.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    x: 400,
+    y: 20
+  })
+  console.log("Green Element has been moved")
+}
+translateOneByOne()
+
+
+function translateAllAtOnce() {
+  Promise.all(moveElement(document.querySelector("#red-circle"), {
```
Comment (qamarfj) [inline javascript/javascript3/week3/visual/main.js:22]: The Promise.all() method takes an iterable of promises as an input. like  Promise.all([promise1,promise2,...]) you can read more about this here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all

File: javascript/javascript3/week3/visual/main.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    x: 20,
+    y: 300
+  }), moveElement(document.querySelector("#blue-circle"), {
+    x: 400,
+    y: 300
+  }), moveElement(document.querySelector("#green-circle"), {
+    x: 400,
+    y: 20
+  }))
+  console.log('All Elements moved')
```
Comment (qamarfj) [inline javascript/javascript3/week3/visual/main.js:32]: this should be in then block.

File: javascript/javascript3/week3/index.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
+// Fetch movies from this api:
+// Create an array of bad movies
+// Creat an array of bad movies since year 2000
+fetch(
+    "https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json"
+  )
+  .then((moviesResponse) => moviesResponse.json())
+  .then(data => {
+    const getBadMovies = data.filter(m => m.rating < 6)
```
Comment (qamarfj) [inline javascript/javascript3/week3/index.js:9]: I would use some descriptive name instead of m.

**Review by qamarfj — COMMENTED** [review]
Hi @Sarulathaanbu ,
You have done really nice work   👍. I have left some comments in code which require just minor changes.

File: javascript/javascript3/week3/visual/main.js (hunk 1 above)
Comment (Sarulathaanbu) [reply javascript/javascript3/week3/visual/main.js:22]: Got it👍🏻 Thanks for pointing it out

File: javascript/javascript3/week3/visual/main.js (hunk 2 above)
Comment (Sarulathaanbu) [reply javascript/javascript3/week3/visual/main.js:32]: yes 👍

File: javascript/javascript3/week3/index.js (hunk 3 above)
Comment (Sarulathaanbu) [reply javascript/javascript3/week3/index.js:9]: Thank you for your feedback 🙂

**Review by qamarfj — APPROVED** [review]
👍

**Pull request merged**